app/jobs/importing/run_daily_imports_job.rb

```diff
@@ -249,5 +250,14 @@ def sync_with_cas
       GrdaWarehouse::Tasks::PushClientsToCas.new.sync!
       @notifier.ping('Pushed Clients to CAS')
     end
+
+    def run_external_lookup_exports
+      return unless RailsDrivers.loaded.include?(:hmis_external_apis)
+      return unless HmisExternalApis::AcHmis::Exporters::ClientExportUploader.can_run?
+
+      HmisExternalApis::AcHmis::UploadClientsJob.perform_later('clients_with_mci_ids_and_address')
```

Comment:
So this is being run twice a day now, right, because it's also scheduled at https://github.com/greenriver/hmis-warehouse/blob/1d0d651c752d2b1a0a4469ba9f5ea74380f6db1a/lib/tasks/import.rake#L12-L13

can you remove from that location?

Reply:
Thanks for that. I had a vague memory it was somewhere, but I didn't search for it correctly.

Reply:
Am I missing something @yakloinsteak , I don't see a diff on that file 🧐

Reply:
Oh I see, so we're continuing to schedule it with the imports. Nope thats cool